resources/views/projects/milestones/create.blade.php

```diff
@@ -0,0 +1,82 @@
+@extends('layouts.app')
```

Comment:
Este comment es en todo el file:

Si tenemos relativamente poca información que agregar aquí, is it worth adding it as a view? or would be prefer a modal instead?

Reply:
Yes, modal! Acabo de ver lo que hiciste para tasks 👍